pages/phases/phone-assessment-interviews/training.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: article
+permalink: /hiring-phases/phone-assessment-interviews/training/
+section: hiring-phases
+sidenav: hiring-phases
+phase: phone-assessment-interviews
+sub-phase-order: 015
+title: Training SMEs to Conduct Interviews
+intro: While phone interviews are being scheduled with applicants, facilitate a mandatory two-hour interview training for all SMEs who will conduct the next round of interviews that explains how to conduct structured interviews and assess an applicant’s proficiency with the required competencies based on their responses.
+---
+
+Before you schedule training for the first round of interviews, divide the SMEs into two groups based on which round of interviews they will conduct. Consider how many applicants passed resume review, as well as any scheduling conflicts you may be aware of at this time.
```

Comment:
add : as soon as possible, send calendar placeholders to the SMEs for the weeks they're expected to give significant time for inteviews.

Reply:
I'm going to add this to the scheduling section instead.